assets/style.css

```diff
@@ -19,9 +19,22 @@ body {
 	justify-content: center;
 	align-items: center;
 	position: absolute;
-	top: 0;
+	top: 5vw;
 	bottom: 0;
 	left: 5;
 	right: 5;
 	padding: 5vw;
 }
+
+#info {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	top: 0;
+	bottom: 75;
```

Comment:
I don't think you want this element to be positioned 75 (unit is missing) from the bottom of the container. That's what [CSS bottom](https://developer.mozilla.org/en-US/docs/Web/CSS/bottom) does. You probably just want to remove this.

```suggestion
```